pkgs/development/libraries/bullet/default.nix

```diff
@@ -4,21 +4,19 @@
 
 stdenv.mkDerivation rec {
   pname = "bullet";
-  version = "2.87";
+  version = "3.17";
 
   src = fetchFromGitHub {
     owner = "bulletphysics";
     repo = "bullet3";
     rev = version;
-    sha256 = "1msp7w3563vb43w70myjmqsdb97kna54dcfa7yvi9l3bvamb92w3";
+    sha256 = "sha256-uQ4X8F8nmagbcFh0KexrmnhHIXFSB3A1CCnjPVeHL3Q=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = lib.optionals stdenv.isLinux [ libGLU libGL freeglut ]
     ++ lib.optionals stdenv.isDarwin [ Cocoa OpenGL ];
 
-  patches = [ ./gwen-narrowing.patch ];
```

Comment:
Could you please remove that file from the repository if it is not required anymore?

Reply:
Actually, this was introduced by 9ab77680b6a56304f04d5c5266663d7c622d5491 and we should test whether this is still required for building on aarch64 or not.

Reply:
`gwen-narrowing.patch` is required for building on aarch64-linux. So please leave it in, as it does not interfere with the x64_64-linux build. Thanks!

Reply:
Changed it. Didn't tested compilation on aarch64 thought (mostly because it's really slow with the qemu emulation, but will leave it run later, when I won't need my computer (if the CI didn't tested it before, I mean)).

Reply:
It appears the roboschool fork still need the patch file, so I kept it.